source/develop/release-management/features/network/managed_mtu_for_vm_networks.html.md

```diff
@@ -206,7 +215,7 @@ might be below the users expectation.
 The MTU property has to be enabled for creating external networks in webadmin,
 the related AddNetworkOnProviderCommand, and during import and synchronization.
 Similar to network name, the MTU of an external network cannot be updated by
-engine.
+webadmin, but the synchronization is required to update the MTU.
```

Comment:
->  but will be updated during synchronization

Reply:
done